client/src/containers/TripView/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -1,38 +1,35 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`<TripView /> should render component with current trip 1`] = `
+exports[`<TripView /> should render component with current trip of 1 day 1`] = `
```

Comment:
Actually, it is strange as in the test definition there is no "1" after the name of the test...
Maybe it is internal and should not be touched...